app/components/conceptAdd/conceptAdd.controller.js

```diff
@@ -10,10 +10,12 @@
 import helpgif from '../../img/help.gif'
 
 ConceptAddController.$inject = 
-	['concept','serverLocales', 'loadClasses', 'loadDataTypes', 'conceptsService', '$location', '$routeParams', 'openmrsNotification'];
+	['concept','serverLocales', 'loadClasses', 'loadDataTypes', 'conceptsService',
+	'$location', '$routeParams', 'openmrsNotification', '$window'];
 	
 export default function ConceptAddController
-		(concept, serverLocales, loadClasses, loadDataTypes, conceptsService, $location, $routeParams, openmrsNotification){
+		(concept, serverLocales, loadClasses, loadDataTypes, conceptsService,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (58)](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2741608991/issues/source?bid=3196969&fileBranchId=3354756#l17)